spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -0,0 +1,490 @@
+                    'Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic spinal cord injury:'
+                    ' MR imaging findings correlated with neurologic outcome--prospective study with 100 consecutive patients.'
+                    ' Radiology 2007;243(3):820-827.'
+                    'Can be computed using AP diameter or other morphometrics (CSA, RL diameter, eccentricity and solidity).'
+                    'Morphometrics are normalized with a reference database where morphometrics from sct_process_segmentation '
+                    'are brought to PAM50 anatomical space using -normalize-PAM50.'
+    )
+
+    mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatoryArguments.add_argument(
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help='Input csv file name in native space. Example: csa.csv.'
+    )
+    mandatoryArguments.add_argument(
+        '-l',
+        metavar=Metavar.file,
+        required=True,
+        help='.nii file with compression labels. Each compression is denoted by a single voxel of value `1`. '
+             'Example: sub-001_T2w_compression_labels.nii.gz'
+    )
+    mandatoryArguments.add_argument(
+        '-i-PAM50',
+        metavar=Metavar.file,
+        required=True,
+        help='Input csv file name in PAM50 space. Example: csa_PAM50.csv.'
+        'Use sct_process_segmentation with -normalize-PAM50 to generate the file.'
+    )
+    mandatoryArguments.add_argument(
+        '-metric',
+        required=False,
+        help='Metric name to normalize in .csv file output from sct_process_segmentation.'
+        '\nChoices: MEAN(area), MEAN(diameter_RL), MEAN(eccentricity), MEAN(solidity)\n',
+        default='MEAN(diameter_AP)',
+        choices=['MEAN(diameter_AP)', 'MEAN(area)', 'MEAN(diameter_RL)', 'MEAN(eccentricity)', 'MEAN(solidity)'],
+        metavar=Metavar.file,
+    )
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        '-file-participants',
+        metavar=Metavar.file,
+        default='participants.tsv',
+        help='participants.tsv file of helathy controls.'
+    )
+    optional.add_argument(
+        '-sex',
+        type=str,
+        choices=['F', 'M'],
+        help='Select sex of healthy subjet to use for normalization. Requires participant.tsv'
+    )
+    optional.add_argument(
+        '-age',
+        type=int,
+        nargs=2,
+        metavar="[0 100]",
+        help='Select age range of healthy subjet to use for normalization. Requires participant.tsv'
+        'E.g.:-age 60 80'
+    )
+    optional.add_argument(
+        '-o',
+        metavar=Metavar.file,
+        help="Output csv file name. If not provided, _compression_metrics suffix is added to the file name provided by -i flag."
+    )
+    optional.add_argument(
+        '-subject',
+        metavar=Metavar.file,
+        help="Name of subject. Default: filename of -i"
```

Comment:
I don't understand this flag. How this subject will be used? for normalization? only with one subject?

Reply:
It is just the subjects name to include in the output csv file. If it is not specified the filename of `-i` argument is used

Reply:
it is not related to the output file, but the column `subject` in the output .csv file like you see in : https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4003#discussion_r1114902193.

Maybe it is not that useful either and I can modify the column with filename instead

Reply:
ahhh! ok gotcha! So, I suggest to replace the column name 'Subject' by 'File', because this is exactly what it is. And I would suggest to not have the 'subject' feature for now.